CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Versions
 ========
 
+2017-11-28
```

Comment:
2017-12-DEV for now

Reply:
You mean literally `2017-12-DEV` as header ?